GetandTake/Program.cs

```diff
@@ -17,6 +19,16 @@
     options.UseCaseSensitivePaths = true;
 });
 
+builder.Services.AddBreadcrumbs(Assembly.GetExecutingAssembly(), options =>
```

Comment:
Well, there was an idea of implementing your own breadcrumbs, but ok, let's stick with this one